web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="custom_user_field" name="{{ field.name }}" data-field-id="{{ field.id }}">
     <span class="custom-user-field-label-wrapper {{#if field.required}}required-field-wrapper{{/if}}">
-        <label class="settings-field-label inline-block" for="{{ field.name }}" class="title">{{ field.name }}</label>
+        <label class="settings-field-label inline-block title" for="{{ field.name }}">{{ field.name }}</label>
```

Comment:
Does this class do anything? We might want to just delete it in a follow-up; it's the kind of really generic name that we try to purge from the codebase.

```
$ git grep [.]title web/styles/
web/styles/lightbox.css:        .title {
web/styles/portico/portico.css:.title {
web/styles/portico/portico.css:a.title {
```